src/fstab-generator/fstab-generator.c

```diff
@@ -567,6 +577,45 @@ static int add_mount(
                         return r;
         }
 
+        if (units_to_start) {
+                r = strv_consume(units_to_start, TAKE_PTR(automount_name) ?: TAKE_PTR(name));
+                if (r < 0)
+                        return r;
+        }
+
+        return 0;
+}
+
+static int start_units(char **units_to_start) {
+        _cleanup_(sd_bus_flush_close_unrefp) sd_bus *bus = NULL;
+        int r;
+
+        if (!units_to_start) {
+                log_debug("No units to start.");
+                return 0;
+        }
+
+        r = bus_connect_system_systemd(&bus);
+        if (r < 0)
+                return log_error_errno(r, "Failed to get D-Bus connection: %m");
+
+        STRV_FOREACH(unit, units_to_start) {
+                _cleanup_(sd_bus_error_free) sd_bus_error error = SD_BUS_ERROR_NULL;
+
+                log_info("Running request %s/start/replace", *unit);
```

Comment:
"Running request" -> "Requesting"?

Reply:
This text appears in a few places and I just copied it here. I agree it should be changed, but I'll do it in a separate commit to do it everywhere.